modules/imaging_browser/jsx/ImagePanel.js

```diff
@@ -73,130 +81,171 @@ var ImagePanelHeadersTable = React.createClass({
+                        this.props.HeaderInfo.XStep + " mm "}
+                {this.props.HeaderInfo.YStep === '' ? ' ' : 'Y: ' +
+                        this.props.HeaderInfo.YStep + " mm "}
+                {this.props.HeaderInfo.ZStep === '' ? ' ' : 'Z: ' +
+                        this.props.HeaderInfo.ZStep + " mm "}
+            </td>
+        </tr>,
+          /* SERIES NUMBER x SNR */
+        <tr>
+          <th className="col-xs-3 info">Series Number</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.SeriesNumber}
+          </td>
+          <th className="col-xs-3 info">SNR</th>
+          <td className="col-xs-3">
+            {this.props.SNR}
+          </td>
+        </tr>,
+          /* SERIES DESCRIPTION x SERIES PROTOCOL */
+        <tr>
+          <th className="col-xs-3 info">Series Description</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.SeriesDescription}
+          </td>
+
+          <th className="col-xs-3 info">Protocol</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.AcquisitionProtocol}
+          </td>
+        </tr>,
+
+          /* ACQUISITION DATE x INSERTED DATE */
+        <tr>
+            <th className="col-xs-3 info">Acquisition Date</th>
+            <td className="col-xs-3">
+                      {this.props.HeaderInfo.AcquisitionDate}
+            </td>
+
+          <th className="col-xs-3 info">Inserted Date</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.InsertedDate}
+          </td>
+        </tr>,
+
+          /* OUTPUT TYPE x SPACE */
+        <tr>
+          <th className="col-xs-3 info">Output Type</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.OutputType}
+          </td>
+
+          <th className="col-xs-3 info">Space</th>
+            <td className="col-xs-3">
+                      {this.props.HeaderInfo.CoordinateSpace}
+            </td>
+        </tr>,
+
+          /* ECHO TIME x REPETITION */
+        <tr>
+          <th className="col-xs-3 info">Echo Time</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.EchoTime} ms
+          </td>
+
+          <th className="col-xs-3 info">Rep Time</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.RepetitionTime} ms
+          </td>
+        </tr>,
+
+        /* NUMBER OF VOLUMES x SLICE THICKNESS */
+        <tr>
+          <th className="col-xs-3 info">Number of volumes</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.NumVolumes} volumes
+          </td>
+
+          <th className="col-xs-3 info">Slice Thick</th>
+          <td className="col-xs-3">
+            {this.props.HeaderInfo.SliceThickness} mm
+          </td>
+        </tr>
+      ];
+
+    // only display DTI algorithm / pipeline rows
+    // if algorithm and pipeline are defined
+    if (this.props.HeaderInfo.Algorithm && this.props.HeaderInfo.Pipeline) {
```

Comment:
@cmadjar could you confirm that this is an accurate assumption?

Reply:
@davidblader Where do you get Algorithm and Pipeline from in the DB?

Reply:
i have no idea. these are just the fields responsible for displaying which algorithm and pipeline are being used. they could either be coming from any of the following queries: 
![image](https://user-images.githubusercontent.com/6571449/42468168-1608fd3a-8382-11e8-8cc8-963680d14ae8.png)

i'm assuming that if both Algorithm & Pipeline are blank, then the remaining table entries are not relevant / valid

Reply:
@davidblader Is it possible that your files table is not up to date? Some fields have been removed, including the Algorithm field I believe

Here is the current version of the files table:
![screen shot 2018-07-09 at 12 35 27 pm](https://user-images.githubusercontent.com/1402456/42470070-6d3c912a-8387-11e8-8593-e525d1d0f1a1.png)

To differentiate native from processed data, I would use the following fields:
SourcePipeline, PipelineDate and OutputType
FYI, for DTIPrep data, OutputType=qc, for native raw data, OutputType=raw

Does that help?

Reply:
my table is up to date, but the Headers Table display in imaging browser i'm guessing is not. as of 20.0 Algorithm and Pipeline are still used. i can't really say for sure where these fields are stored in the DB since CCNA has no examples where these fields get populated. 
i can update these fields in a separate PR if you tell me what should be displayed and from which DB columns?

Reply:
hahaha, I don't think my screenshot from yesterday was useful :P. Uploaded the wrong screenshot...

Anyway, I am pretty sure Algorithm is useless and should be removed. For the Pipeline one, it is possible that it is reading the SourcePipeline field of the files table. Is your PR in a state to be tested? If so, I can pull it on my sandbox and check with my data (I have processed data on my sandbox)